docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -92,7 +143,40 @@ locations:
       registry: 764506304434.dkr.ecr.us-east-1.amazonaws.com/branch-deployment-agent
 ```
 
-### Step 4.2: Configure GitHub Action secrets
+### Step 2.3: Configure GitHub Action secrets
+
+In this step, you'll configure the secrets used by the Branch Deployments GitHub Action. The steps and required secrets vary a bit depending on the type of deployment you're using.
+
+Select your deployment type below to view instructions.
+
+<details>
+  <summary><strong>SERVERLESS DEPLOYMENTS</strong></summary>
+
+1. In the repository, click the **Settings** tab.
+2. In the **Security** section of the sidebar, click **Secrets > Actions**.
+3. Click **New repository secret**.
+4. In the **Name** field, enter the name of the secret. For example, `DAGSTER_CLOUD_API_TOKEN`
+5. In the **Value** field, paste the value of the secret.
+6. Click **Add secret**.
+
+Repeat steps 3-6 for each of the following:
+
+- `DAGSTER_CLOUD_API_TOKEN` - The Serverless agent token. Refer to the [Managing user and agent tokens guide](/dagster-cloud/account/managing-user-agent-tokens) for more info.
```

Comment:
If a Serverless user is doing this post-onboarding (and the GH app setup is no longer available), should they generate a new agent token or use an existing agent token?